src/main/java/pl/magisterka/dominikszojda/repository/DogRepository.java

```diff
@@ -13,11 +13,13 @@ public interface DogRepository extends CrudRepository<DogEntity, UUID>{
 
     Optional<DogEntity> findByNameAndAge(String name, int age);
 
+    DogEntity assignDogToOwner(UUID dogId, UUID ownerId);
```

Comment:
To będzie cokolwiek robić/działać? Nie mogę znaleźć nic o `assign`. Jeżeli chcesz mieć taką metodę to musisz użyć relacji, czyli to będzie coś w stylu:

```java
DogEntity dog = ...;
DogOwnerEntity owner = ownerRepo.findById(ownerId);
dog.owner = owner;
dogRepo.save(dog);
```